.github/workflows/ai-validation.yml

```
@@ -36,16 +36,40 @@
       - name: Checkout code
         uses: actions/checkout@v4
 
+      - name: Read package.json
+        id: package
+        run: |
+          NODE_VERSION=$(jq -r '.engines.node' package.json)
+      - name: Read package.json
+        id: package
```

actionlint (1.7.4)

step ID "package" duplicates. previously defined at line:40,col:13. step ID must be unique within a job. note that step ID is case insensitive

(id)